resmio-api/resmio-api.php

```diff
@@ -288,22 +288,22 @@ function resmio_api_data_update() {
 	else:
 	?>
 	<script type = "text/javascript" language = "javascript">
-		$( document ).ready(function {
+		jQuery( document ).ready(function() {
```

Comment:
In WordPress jQuery is implemented by default but you have to use 'jQuery' instead of '$'